kernel/kernel-api/src/main/java/io/delta/kernel/internal/tablefeatures/TableFeatures.java

```diff
@@ -39,6 +40,9 @@
 /** Contains utility methods related to the Delta table feature support in protocol. */
 public class TableFeatures {
 
+  /** The prefix for setting an override of a feature option in the */
+  public static String PROPERTIES_FEATURE_OVERRIDE_PREFIX = "delta.feature.";
```

Comment:
are these properties persisted to the delta log? (I don't think so) -- maybe add where this is enforced? important detail? wdyt?